<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewOrderForm</class>
 <widget class="QWidget" name="AddNewOrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оформление нового заказа</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата оформления заказа:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Идентификатор заказа:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>711</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Список заказанных товаров или ремонтных работ: </string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>691</width>
      <height>321</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
   </widget>
  </widget>
  <widget class="QDateEdit" name="orderRegDate_dateEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="orderStatus_lineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>711</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Персональные и контактные данные заказчика:  </string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fullName_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Адресс проживания:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="livingAdress_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер телефона: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoheNumber_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес доставки: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deliveryAdress_lineEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Год рождения:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="birhtDay_dateEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="orderID_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Статус заказа:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addItem_pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить товар/услугу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addOrder_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>670</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить заказ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
